tests/template/logparser_data/opal3.txt

```diff
@@ -0,0 +1,109 @@
+OPAL{0}>                ____  _____       ___ 
+OPAL{0}>               / __ \|  __ \ /\   | | 
+OPAL{0}>              | |  | | |__) /  \  | |
+OPAL{0}>              | |  | |  ___/ /\ \ | |
+OPAL{0}>              | |__| | |  / ____ \| |____
+OPAL{0}>               \____/|_| /_/    \_\______|
+OPAL{0}> 
+OPAL{0}> This is OPAL (Object Oriented Parallel Accelerator Library) Version 2022.1.0
+OPAL{0}>                              git rev. unknown
+OPAL{0}> 
+OPAL{0}> 
+OPAL{0}>                      (c) PSI, http://amas.web.psi.ch
+OPAL{0}> 
+OPAL{0}> 
+OPAL{0}> The optimiser (former opt-Pilot) is integrated 
+OPAL{0}> 
+OPAL{0}> Please send cookies, goodies or other motivations (wine and beer ... ) 
+OPAL{0}> to the OPAL developers opal@lists.psi.ch
+OPAL{0}> 
+OPAL{0}> Time: 22:35:26 date: 26/01/2024
+OPAL{0}> 
+OPAL{0}> * Reading input stream 'opal.in'
+Error{1}> S1: No filename for a field map given. Will assume the default map "1DPROFILE1-DEFAULT".
```

Comment:
above this @moellep ?

Reply:
The existing opal parser has a special case to ignore 1DPROFILE1-DEFAULT